src/utils/env.ts

```diff
@@ -7,17 +7,17 @@ import type { ParsedSpace } from "../types";
  * can be present using comma as separator.
  *
  * @param input - The SPACES environment variable string
- * @returns A boolean indicating the validity of the input format
```

Comment:
This may have been done in a different commit. I just realized the annotations weren't completely right